lib/src/serializable.dart

```diff
@@ -82,8 +85,13 @@ class SerializableBus {
 
   Bridge get bridge => _bridge;
   set bridge(Bridge bridge) {
+    if (_bridge != null) {
+      _bridge.dispose();
+    }
+
     _bridge = bridge;
-    _bridge.apiCallReceived.listen(_handleApiCall);
+    _bridge.manageStreamSubscription(
```

Comment:
Probably need to ensure that `bridge` isn't being set to null here, otherwise this would throw (would also prevent a failure below when reset() is called)